doc/reference/configuration/configuration_reference.rst

```diff
@@ -2988,6 +2988,73 @@ The ``memtx`` section is used to configure parameters related to the :ref:`memtx
     | Default: box.NULL
     | Environment variable: TT_MEMTX_SORT_THREADS
 
+
+
+
+..  _configuration_reference_metrics:
+
+metrics
+-------
+
+The ``metrics`` section defines configuration parameters for :ref:`metrics <monitoring>`.
+
+..  NOTE::
+
+    ``metrics`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`metrics.exclude <configuration_reference_metrics_exclude>`
+-   :ref:`metrics.include <configuration_reference_metrics_include>`
+-   :ref:`metrics.labels <configuration_reference_metrics_labels>`
+
+
+.. _configuration_reference_metrics_exclude:
+
+..  confval:: metrics.exclude
+
+    Metrics to turn off.
+    This option accepts the same values as the ``exclude`` configuration parameter passed to :ref:`metrics.cfg() <metrics_cfg>`.
+
+    **Example**
+
+    ..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics/config.yaml
+        :start-at: metrics:
+        :end-at: instance_name
+        :language: yaml
+        :dedent:
+
+    |
+    | Type: array
+    | Default: nil
+    | Environment variable: TT_METRICS_EXCLUDE
+
+
+.. _configuration_reference_metrics_include:
+
+..  confval:: metrics.include
+
+    Metrics to turn on.
+    This option accepts the same values as the ``include`` configuration parameter passed to :ref:`metrics.cfg() <metrics_cfg>`.
+
+    |
+    | Type: array
+    | Default: nil
```

Comment:
Actually it's `['all']` array (even though behavior with `nil` is the same since `metrics.cfg` defaults will treat `nil` as `['all']`).

https://github.com/tarantool/tarantool/blob/0b014ad3693bf41ffd25a27186fad8a53cc37205/src/box/lua/config/applier/box_cfg.lua#L364